templates/Start/vouchers.html.twig

```diff
@@ -31,7 +31,7 @@
                             </div>
                         {% else %}
                             <div class="alert alert-success">
-                                <p class="voucher-box">{{ voucher.code }}</p>
+                                <p class="voucher-box"><a href="{{ url('register_voucher', {'voucher': voucher.code}) }}" target="_blank">{{ voucher.code }}</a></p>
```

Comment:
We should add a "click to copy link" button and not use a link.

Reply:
Agreed, but I don't have clue how to do that. Would you be ok with doing this later in a separate step?